docs/source/platform/schema-validation.mdx

```diff
@@ -250,7 +250,7 @@ To install the Apollo Engine integration on GitHub, go to [https://github.com/ap
 
 ### Posting a comment to your PRs
 
-For teams using GitHub Enterprise, Bitbucket, and other source control tools, we recommend setting up your CI to post a comment on your PRs with the results of schema validation. Surfacing schema diffs and breaking changes directly in your PR will speed up your review workflow by saving you the time of searching your CI logs to check why validation didn't pass.
+For teams using GitHub Enterprise, Bitbucket, and other source control tools, we recommend setting up your CI to post a comment on your PRs with the results of schema validation. Surfacing schema diffs and breaking changes directly in your PR will speed up your review workflow by saving you the time of searching your CI logs to check why validation didn't pass.
```

Comment:
I don't understand why there's a diff on that space

Reply:
me neither